wordpress-mobile/WordPress-Android
# Feature/deep linking improvements

Fixes #6313

To test:

Go to the following sections of WordPress.com with the app installed and logged in

Reader and verify when clicking the app banner it goes to the Reader
Stats and verify when clicking the app banner it goes to stats
Editor and verify when clicking the app banner it goes to the editor
Notifications and verify when clicking the app banner it goes to notifications
